src/test/java/org/dita/dost/module/KeyrefModuleTest.java

```diff
@@ -363,4 +364,38 @@ public void testWalkMapAndRewriteKeydefHrefKeepAnchor() throws URISyntaxExceptio
 						.where(Predicates.attributeEq(ATTRIBUTE_NAME_HREF, "topic-2.dita#abc")))
 				.exists());
 	}
+
+    @Test
+    public void testNoNPEPeerMap() throws URISyntaxException, XPathException {
+        inputMapFileInfo = new Builder()
+                .uri(create("test4.ditamap"))
+                .src(new File(baseDir, "src" + File.separator + "test4.ditamap").toURI())
+                .result(new File(baseDir, "src" + File.separator + "test4.ditamap").toURI())
+                .format("ditamap")
+                .isInput(true)
+                .build();
+        job.add(inputMapFileInfo);
+        final XdmNode act = parse(inputMapFileInfo.src);
+        Map<String, KeyDef> defsMap = new HashMap<>();
+        defsMap.put("Tool", new KeyDef("Tool", null, "local", "dita", inputMapFileInfo.src, null));
+        
+        KeyScope submap1 = new KeyScope("submap", " mapref[6]map[6].submap", defsMap, Arrays.asList(new KeyScope[] {}));
```

Comment:
Should the second argument have a leading space?

Reply:
What second argument?

Reply:
On line 382, the second argument to KeyScope() is `" mapref[6]map[6].submap"` which is different than line 383, so I was curious about the difference.

Reply:
I removed that extra space as well.